recipes/datapane/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set name = "datapane" %}
+{% set version = "0.6.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 398fdf46259e6e7485e01ddb817fe27158797120346464fbe476c80ac13f8e5b
+  
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - datapane = datapane.client.__main__:main
+    - dp-runner = datapane.runner.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - poetry
+  run:
+    - python
+    - dataclasses
+    - altair >=4.0.0,<5.0.0
+    - bokeh >=2.0.0,<3.0.0
+    - click >=7.0.0,<8.0.0
+    - click-spinner >=0.1.8,<0.2.0
+    - colorlog >=4.1.0,<5.0.0
+    - dacite >=1.5.0,<2.0.0
+    - flit-core >=2.3.0,<2.4.0
+    - folium >=0.11.0,<0.12.0
+    - importlib_resources >=3.0.0,<4.0.0
+    - jinja2 >=2.11.1,<3.0.0
+    - jsonschema >=3.2.0,<4.0.0
+    - matplotlib >=3.1.0,<4.0.0
+    - munch >=2.5.0,<3.0.0
+    - nbconvert >=5.6.1,<6.0.0
+    - numpy >=1.18.0,<2.0.0
+    - packaging >=20.3,<21.0
+    - pandas >=1.0.1,<2.0.0
+    - plotly >=4.8.1,<5.0.0
+    - pyarrow >=0.17.0,<0.18.0
+    - pyyaml >=5.3.0,<6.0.0
+    - requests >=2.20.0,<3.0.0
+    - ruamel.yaml >=0.16.5,<0.17.0
+    - stringcase >=1.2.0,<2.0.0
+    - tabulate >=0.8.7,<0.9.0
+    - toolz >=0.10.0,<0.11.0
+    - validators >=0.15.0,<0.16.0
+
+test:
+  imports:
+    - datapane
+    - datapane.client
+    - datapane.client.api
+    - datapane.client.scripts
+    - datapane.common
+    - datapane.resources
+    - datapane.resources.local_report
+    - datapane.resources.scaffold
+    - datapane.runner
+  commands:
+    - datapane --help
+    - datapane --version
+    - dp-runner --help
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://datapane.com
+  license: Proprietary
```

Comment:
We can't distribute this a the license file in the tarball states that you can't do anything with this package:
```
Copyright (c) 2015-2020, StackHut Limited (trading as Datapane).
All rights reserved.
```
At the very least conda-forge, NumFOCUS and Anaconda Inc need to have permission to redistribute the package without restrictions.

Reply:
Thanks for the feedback. No problem - we were in the process of moving to Apache-2.0 internally anyway so I just implemented that over the weekend and repushed - cheers!